src/main/java/org/inferred/freebuilder/processor/Processor.java

```diff
@@ -93,6 +108,9 @@ public synchronized void init(ProcessingEnvironment processingEnv) {
 
   @Override
   public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment roundEnv) {
+    if (analyser == null) {
+      return false;
```

Comment:
Why not add a comment explaining when this could happen?
Not everybody reading the code is a star programmer like you who gets it on their own.